portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +26,9 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Nzinga!  </h1>");
+    response.getWriter().println("Hello Nzinga!");
+    
```

Comment:
nit (optional fix): the blank lines here are not something I usually see in a Java method.